client/src/components/Navbar.js

```diff
@@ -12,9 +12,9 @@ const Navbar = (props) => {
  
   const StyledButton = styled(Button)({
     color: '#757575',
-    padding: '0rem 0rem',
+    padding: '10px 0',
```

Comment:
i agree with you that `6px` looks better

Reply:
Font size has been updated!